modules/gitops-in-built-permissions.adoc

```diff
@@ -0,0 +1,18 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/configuring-a-cluster-to-use-gitops.adoc
+
+[id="in-built-permissions_{context}"]
+= In-built permissions for Argo CD
+
+This section lists the permissions that are granted to ArgoCD to manage specific cluster-scoped resources which include platform operators, optional OLM operators, and user management. Note that ArgoCD is not granted `cluster-admin`.
+
+.Permissions granted to Argo CD
+|==========================
+|Resource groups|What does it configure for a user or an administrator
+|operators.coreos.com      |Optional operators managed by OLM.   
+|user.openshift.io , rbac.authorization.k8s.io |Groups, Users, and their permissions.
+|config.openshift.io       |Control plane operators managed by CVO used to configure cluster-wide build configuration, registry configuration, scheduler policies, and so on.  
```

Comment:
remove `and so on` as per ISG: Use "and so on" when you list a clear sequence of elements, such as "1, 2, 3, and so on" or
"Monday, Tuesday, Wednesday, and so on". When you list items that do not form a clear sequence, use
appropriate descriptive wording, such as "device drivers, firmware, and other code".